sentinelhub/config.py

```diff
@@ -217,28 +212,52 @@ def __repr__(self) -> str:
         return "\n  ".join(repr_list).strip(",") + "\n)"
 
     @property
-    def _global_instance(self) -> _SHConfig:
+    def _global_instance(self) -> "SHConfig":
         """Uses a class attribute to store a global instance of a class with config parameters."""
         if SHConfig._instance is None:
-            SHConfig._instance = SHConfig._SHConfig()
+            SHConfig._instance = SHConfig.load_configuration(self.get_config_location())
         return SHConfig._instance
 
-    def save(self) -> None:
+    @classmethod
+    def load_configuration(cls, filename: str) -> "SHConfig":
```

Comment:
I'm wondering if we should even shorten this method name into `load`? As it is a class method it is obvious what `SHConfig.load(path)` will load.

Reply:
renamed